prod.tfvars

```diff
@@ -1,7 +1,11 @@
 sa_account_tier       = "Premium"
 sa_replication_type   = "GRS"
-vnet_address_space    = "10.50.13.128/25"
-snet01_address_prefix = "10.50.13.128/28"
-snet02_address_prefix = "10.50.13.192/26"
+vnet_address_space    = "10.101.3.0/24"
+snet01_address_prefix = "10.101.3.0/26"    #pre-video-edit-vm-prod
+snet02_address_prefix = "10.101.3.64/26"   #pre-privatelink-vm-prod
+snet01_address_prefix = "10.101.3.128/26"  #pre-bastion-prod
```

Comment:
avoid using numbers in variables just use the subnet name it'll make it easier to read then you can remove the comment

Reply:
modified